docs/src/main/sphinx/connector/optimize.fragment

```diff
@@ -13,10 +13,10 @@ ALTER TABLE test_table EXECUTE optimize
 ```
 
 The following statement merges files in a table that are
-under 10 megabytes in size:
```

Comment:
not sure about line no 9 in that file. It says that the default is 100MB. Should I change that to 128MB?

Reply:
No. That is correct. Current default is 100MB.